### PR TITLE
Integration of Post with Comment component

### LINKED ISSUE #27: Create Post UI
API : **/post/create**

### LINKED ISSUE #31: Implement UI of feedpage
- Post component
- Navbar component

### LINKED ISSUE #66: Comment component
- [x] Create commenet
- [ ] Delete comment if authorized

### PR DESCRIPTION
**Pull request by aitikgupta**
Resolves https://github.com/Open-Sourced-Olaf/SocioMark/issues/27
Resolves https://github.com/Open-Sourced-Olaf/SocioMark/issues/31
Resolves https://github.com/Open-Sourced-Olaf/SocioMark/issues/66

### CODE REVIEW

File: frontend/src/Components/UserProfile.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React, { useContext } from "react";
-import { AuthContext, AuthProvider } from "../Context/AuthContext";
+import { AuthContext } from "../Context/AuthContext";
 
 export default function UserFeed() {
   const { value1, value2 } = useContext(AuthContext);
```
Comment (bodhisha) [inline frontend/src/Components/UserProfile.js]: Need to change value1,value2 to user and token to here also right?

**Review by bodhisha — APPROVED** [review]

File: frontend/src/Components/UserProfile.js (hunk 1 above)
Comment (aitikgupta) [reply frontend/src/Components/UserProfile.js]: Yes, thanks for catching that!

**Comment by aitikgupta** [comment]
@bodhisha I added one more commit, can you review that as well? :D

**Review by bodhisha — APPROVED** [review]
Awesome! 🎉

**Pull request merged**